tdrs-frontend/src/utils/index.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+let memo = null
+
+const exports = {
+  get axiosInstance() {
+    if (memo) return memo
+    memo = axios.create({
+      withCredentials: true,
```

Comment:
We had some difficulty testing this, it could possibly be refactored, but we are delaying that to a future ticket since this works. We think it is possible the csrf issue is a configuration issue on the frontend ticket. The goal of adding this was to unblock the project while we pursued this